addons/hr_attendance/models/ir_websocket.py

```diff
@@ -0,0 +1,27 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import re
+
+from odoo import models
+
+
+class IrWebsocket(models.AbstractModel):
+    """Override to handle hr_employee specific features (presence in particular)."""
+
+    _inherit = "ir.websocket"
+
+    def _build_bus_channel_list(self, channels):
+        employee_channel_ids = []
+        remaining_channels = []
+        has_employee_public_access = self.env["hr.employee.public"].has_access("read")
+
+        for ch in channels:
+            if isinstance(ch, str):
+                match = re.findall(r"^hr\.employee_(\d+)$", ch)
```

Comment:
Use ``channels.remove(channel)`` instead of reconstructing the ``remaining_channels`` list.